google-cloud-tools-plugin/src/com/google/gct/idea/settings/ImportSettings.java

```diff
@@ -55,6 +55,7 @@
    */
   public static void doImport(String path) {
     final File saveFile = new File(path);
+    ZipFile saveZipFile = null;
```

Comment:
Seems like I have to put it there so that I can use it inside 'finally' to close the file. Is there some other elegant way?